Services/AuthenticationService/Commands/LoginCommand.cs

```diff
@@ -49,7 +49,15 @@ public async Task<string> Execute(UserEmailPassword data)
                 throw new Exception("User not found =(");

```

Comment:
Why don't custom exc?

Reply:
> Why don't custom exc?

Because they suck right now, I updated them in another pr

Reply:
> Why don't custom exc?

Answered above